django/db/models/expressions.py

```diff
@@ -10,6 +10,20 @@
 from django.utils.functional import cached_property
 
 
+class SQLiteNumericMixin:
+    """
+    Some expressions with output_field=DecimalField() must be casted to
+    numeric to be properly filtered.
+    See https://code.djangoproject.com/ticket/18247.
```

Comment:
Generally we would rather expand the docstring with sufficient explanation rather than include a ticket reference (or at least keep the mention limited to a simple "(#18247)" as in tests).